app/web/features/dashboard/locales/en.json

```diff
@@ -1,33 +1,35 @@
-    "community_builder_form_link": "https://couchers.org/community-builder-form",
-    "my_events": "My events"
+  "please_complete_profile": "Please complete your profile; and ensure you show up in search:",
+  "fill_in_who_i_am": "1. Fill in your \"Who I am\" section with a few sentences about yourself",
+  "upload_photo": "2. Upload a photo (by clicking on the avatar on the edit page)",
+  "click_here_to_edit": "Edit your profile",
+  "dont_you_hate": "Don't you hate how other platforms have empty \"ghost\" profiles with no information. Please don't ghost us! 👻👻👻",
+  "password_text_1": "You can set a password so you don't need check your email each time you log in",
+  "password_text_link": "Account Settings",
+  "your_communities_heading": "Your communities",
+  "all_communities_heading": "All communities",
+  "all_communities_link": "Browse all communities",
+  "no_community": "You are not currently in a community.",
+  "no_sub_communities": "No sub-communities.",
+  "load_more": "Load more",
+  "welcome": "Welcome to the Couchers.org Beta!",
+  "new_pill": "New",
+  "updates_title": "Feature Updates",
+  "updates_pill": "Software",
+  "last_update": "Last updated: 14th October, 2021",
+  "updates_markdown": "\n\n* Events: events happening in your communities now show up on the dashboard\n* Handbook: we started on the Support Handbook with an initial help page\n* Chat: you can now press enter to send a message\n* We added page titles so that you can now disambiguate between different tabs\n* Profile: the save button now floats on the screen so it's visible at all times\n* Events: fixed a bug that prevented editing an ongoing event\n\n## Mobile applications for iOS and Android\n\nWe've started building mobile apps for Couchers.org, and will be giving updates on the progress in the coming months. If you're a developer with experience using React Native and are interested in contributing to this effort, please complete the [contributor form](https://couchers.org/contribute) and let us know!\n\n## Events and community builders\n\nWe've been working hard to build the events feature for you and your community, as well as other functionality for Community Builders to help build their local community. If you'd like to help out in building the couch surfing community near you through Couchers.org, please let us know by filling in the [Community Builder form](https://couchers.org/community-builder-form)!\n\n*The Beta is still being worked on at a rapid rate, and things will continue to change quickly for the foreseeable future, so check back often for updates.*\n\n# Bugs\n\nBugs are to be expected, as we are still in a phase of rapid development. We appreciate you being here to help with that. If you notice any bugs, please report them with the red button in the top right corner.\n\nPlease discuss anything else about the app on the [Community Forum](https://community.couchers.org/).",
```

Comment:
I feel like this "updates_markdown" should stay in the constant file too (or maybe even in its own markdown file), since it's constantly being updated by our team and so I'd classify it as dynamic content (and as if it comes from the backend). It's a bit of hack we are putting it in the frontend tbh...

Reply:
Please check this one with @lucaslcode
I think he creates the en.json files.
My opinion is that it is ok to keep it in the translation file.

Reply:
Yes, keep this in constants as it gets changed frequently and won't be translated

Reply:
Fine! Resolved with caf96e8066bbafd544965509e4ac53ce038d0f60